common/presenters/population-table/location-meta-to-spaces-link.js

```diff
@@ -0,0 +1,17 @@
+const i18n = require('../../../config/i18n')
+
+const spacesFormatter = function (count) {
+  if (count === undefined) {
+    return i18n.t('population::add_space')
+  }
+
+  return i18n.t('population::spaces_with_count', { count })
+}
+
+module.exports = function locationToMetaToSpacesLink({
+  hrefPrefix = '',
+  space,
+  id,
```

Comment:
Is this being used? I would have expected eslint to pick this up...?

Reply:
It's now removed for this PR